terasoluna-gfw-parent/pom.xml

```diff
@@ -208,13 +208,8 @@
 
       <!-- == Begin Dozer== -->
       <dependency>
-        <groupId>net.sf.dozer</groupId>
-        <artifactId>dozer</artifactId>
-        <version>${dozer.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>net.sf.dozer</groupId>
-        <artifactId>dozer-spring</artifactId>
```

Comment:
Why do you drop dozer-spring?
I think it need.

In dozer 6.2.x, [dozer-spring4](https://github.com/DozerMapper/dozer/tree/master/dozer-integrations/dozer-spring-support/dozer-spring4) has been provided.

Reply:
@kazuki43zoo Thanks for your comments. I re-added `dozer-spring4`.